// Error Handlling
let a = 4;
let b = 5;
console.log(`a${a}`);
console.log(`a+b  = ${a+b}`);

try{

    console.log(`a+b  = ${a+c}`); // error
}
catch(err){ // err = error objects 
console.log(err);
}
console.log(`a+b  = ${a+b}`);
console.log(`a+b  = ${a+b}`);
console.log(`a+b  = ${a+b}`);
console.log(`a+b  = ${a+b}`);
console.log(`a+b  = ${a+b}`);
